internal/backend/svc/banner.go

```diff
@@ -3,82 +3,139 @@ package svc
 import (
 	_ "embed"
 	"fmt"
+	"io"
 	"os"
+	"strings"
 	"text/template"
 
 	"github.com/fatih/color"
+	"go.uber.org/fx"
 
 	"go.autokitteh.dev/autokitteh/internal/backend/configset"
+	"go.autokitteh.dev/autokitteh/internal/backend/httpsvc"
+	"go.autokitteh.dev/autokitteh/internal/backend/temporalclient"
+	"go.autokitteh.dev/autokitteh/internal/backend/webplatform"
 	"go.autokitteh.dev/autokitteh/internal/kittehs"
 	"go.autokitteh.dev/autokitteh/internal/version"
 )
 
 type bannerConfig struct {
-	Show bool `koanf:"show"`
+	Show    bool `koanf:"show"`
+	Minimal bool `koanf:"minimal"`
```

Comment:
First document this config section in "Server Config" in Docusaurus, as requested in the past. Then we could discuss adding more config values.